swift/apple/FirezoneKit/Sources/FirezoneKit/Managers/SystemExtensionManager.swift

```diff
@@ -7,20 +7,21 @@
 #if os(macOS)
 import SystemExtensions
 
-enum SystemExtensionError: Error {
-  case UnexpectedResult(result: OSSystemExtensionRequest.Result)
-  case NeedsUserApproval
+// Maintain a static handle to the extension manager for tracking the state of the extension activation.
```

Comment:
Nit: These docs should be on the `static` field, not the class.

Reply:
Why can't the `static` field not just be on the class itself? Feels weird to have a separate type for it that otherwise doesn't do anything.

Reply:
In a typical singleton pattern, the global instance is just defined on the type itself.

Reply:
Whoops, good catch. Simplified.